SampleApps/WebView2_WinUI3_Sample/readme.md

```diff
@@ -0,0 +1,50 @@
+# Introduction
+This sample shows off using a WebView2 control in a WinUi 3 Windows SDK Packaged application.
+
+It also optionaly shows how you would update the application to ship with a fixed WebView2 version instead of using the version installed and running on the Windows computer.
```

Comment:
```suggestion
It also optionaly shows how you would update the application to ship with a fixed version of the WebView2 Runtime instead of using the version that's installed and running on the user's computer.
```